src/renderer/widgets/Transfer/model/form-model.ts

```diff
@@ -91,129 +90,64 @@ const $transferForm = createForm<FormParams>({
     account: {
       init: {} as Account,
       rules: [
-        {
-          name: 'noProxyFee',
-          source: combine({
+        TransferRules.account.noProxyFee(
+          combine({
```

Comment:
I guess this is why we have `.getState()` inside validation function 
Let's schedule a call to resolve using `.getState()`, it can lead to unexpected problems in future